### PR TITLE
add arbitrary change

### PR DESCRIPTION
**Pull request by pxr687**

### CODE REVIEW

File: dunning_kruger.Rmd [hunk 1]
```diff
@@ -103,3 +103,7 @@ df
 ```{python}
 df.groupby('actual_quantile', observed=True)['difference'].mean()
 ```
+
+```{python}
+# arbitrary change
```
Comment (matthew-brett) [inline dunning_kruger.Rmd:108]: This change does look a little arbitrary.  Can you think of anything less arbitrary to add?

**Review by matthew-brett — COMMENTED** [review]
A comment.

File: dunning_kruger.Rmd (hunk 1 above)
Comment (pxr687) [reply dunning_kruger.Rmd:108]: Ok I'll have a think.

File: dunning_kruger.Rmd [hunk 2]
```diff
@@ -103,3 +103,25 @@ df
 ```{python}
 df.groupby('actual_quantile', observed=True)['difference'].mean()
 ```
+
+```{python}
+# add some randomnss to the the perceived percentiles
```
Comment (matthew-brett) [inline dunning_kruger.Rmd:108]:
```suggestion
# add some randomness to the the perceived percentiles
```

File: dunning_kruger.Rmd [hunk 3]
```diff
@@ -103,3 +103,25 @@ df
...
+df.groupby('actual_quantile', observed=True)['difference_w_randomness'].mean()
+```
+
+```{python}
+
+```
+
+```{python}
+
+```
```
Comment (matthew-brett) [inline dunning_kruger.Rmd:127]:
```suggestion
```

Delete empty cells.

**Review by matthew-brett — COMMENTED** [review]
Nearly there.

**Review by pxr687 — COMMENTED** [review]
Looks good to me!

**Pull request merged**